frontend/styles/how-it-work.css

```diff
@@ -0,0 +1,5 @@
+.num-component > div {
+  z-index: 2;
+  position: relative;
+  background-color: #fff;
```

Comment:
Avoid using any colors outside the tailwind color theme as this will be so hard to change later
and you don't need to set the background here anyway